contracts/StabilityBoardSigner.sol

```diff
@@ -0,0 +1,11 @@
+/* allows tx to execute if 50% +1 vote of active signers signed */
+pragma solidity 0.4.24;
+import "./generic/MultiSig.sol";
+
+
+contract StabilityBoardSigner is MultiSig {
```

Comment:
Suggest naming this `StabilityBoardProxy` as this acts as a proxy for the intentions of the actual stability board (a collection of people).